sdcm/monitorstack/__init__.py

```diff
@@ -465,6 +465,7 @@ def remove_sct_metrics(metric):
             -s {monitoring_dockers_dir}/config/scylla_servers.yml \
             -d {monitoring_stack_data_dir} -v {scylla_version} \
             -b '-storage.tsdb.retention.time=100y' \
+            -b '--web.enable-remote-write-receiver' \
```

Comment:
Is monitoring stack's prometheus port closed from outside? This could be an issue if not.

Reply:
By the original design of how networking is setup it should be closed

Reply:
I've removed this part from the code